patches/0073-libavfi-dnn-add-LibTorch-as-one-of-DNN-backend.patch

```diff
@@ -658,7 +674,10 @@ index 0000000000..4f1395414c
 +            av_frame_free(&item->out_frame);
 +            av_freep(&item);
 +        }
++        ff_queue_destroy(th_model->task_queue);
 +    }
++    delete th_model->jit_model;
++    av_opt_free(&th_model->ctx);
```

Comment:
Is it not freed in the common path ?

Reply:
No it is not freed. Valgrind shows a memory leak about opt_ctx. It is inner opt_ctx, so it is not freed in common path.